src/lib.rs

```diff
@@ -44,6 +44,9 @@ pub enum Tee {
     /// Hygon DCU (Deep Computing Unit)
     HygonDcu,
 
+    // Trusted Platform Module
+    Tpm,
```

Comment:
Is the idea that this will be a non-confidential TPM, or a TPM backed by confidential attestation, or generic over both cases, or something else?

Reply:
It can be both. Generally, we could also want to perform attestation using hardware TPMs, even if this isn't confidential.